packages/react-components/src/components/Dialpad/Dialpad.tsx

```diff
@@ -176,20 +184,33 @@ const DtmfTones: DtmfTone[] = [
 /* @conditional-compile-remove(dialpad) */
 const DialpadContainer = (props: {
   placeholderText: string;
+  deleteButtonAriaLabel: string;
```

Comment:
Yeah deleteButtonAriaLabel is used for the  "ariaLabel" prop for IconButton, 
<IconButton
                ariaLabel={props.deleteButtonAriaLabel}
                onClick={deleteNumbers}
                style={{ color: 'black' }}
                iconProps={{ iconName: 'BackSpace' }}
              />
and also deleteButtonAriaLabel is within string.json.
"dialpad": {
    "placeholderText": "Enter a number",
    "deleteButtonAriaLabel": "Delete"
  },
But yeah for dialpad container it should be strings: DialpadStrings instead of defined separately, will update that

Reply:
Updated